internal/binutils/binutils.go

```diff
@@ -140,7 +144,72 @@ func initTools(b *binrep, config string) {
 		b.addr2line, b.addr2lineFound = findExe("gaddr2line", append(paths["addr2line"], defaultPath...))
 	}
 	b.nm, b.nmFound = findExe("nm", append(paths["nm"], defaultPath...))
-	b.objdump, b.objdumpFound = findExe("objdump", append(paths["objdump"], defaultPath...))
+	// Look for llvm-objdump and objdump
+	b.objdump, b.llvmObjdumpFound, b.objdumpFound = FindObjdump(append(paths["llvm-objdump"], append(paths["objdump"], append(paths["gobjdump"], defaultPath...)...)...))
```

Comment:
For addr2line above we use paths[addr2line] for both addr2line and gaddr2line cases. We should probably be consistent with that here.

Reply:
Done.